### ISSUE TITLE
v3.0.0-alpha.0 - shows the following error when getstorybook is run on a project

### ISSUE THREAD

**Issue by Nosherwan** [storybooks/storybook#1053]
getstorybook - the simplest way to add a storybook to your project. 

 • Detecting project type. ✓
 • Adding storybook support to your "Webpack React" app. ✓
 • Preparing to install dependencies. ✓

npm ERR! Darwin 16.6.0
npm ERR! argv "/Users/nosh/.nvm/versions/node/v7.3.0/bin/node" "/Users/nosh/.nvm/versions/node/v7.3.0/bin/npm" "install"
npm ERR! node v7.3.0
npm ERR! npm  v3.10.10
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: @storybook/react@^2.21.0
npm ERR! notarget Valid install targets:
npm ERR! notarget 3.0.0-alpha.0
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'sws-storybook-app'
npm ERR! notarget 

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/nosh/simplywallst/sws-storybook-app/npm-debug.log

 • Installing dependencies. ✖

     An error occurred while installing dependencies.

**Comment by Nosherwan** [comment]
Please suggest how can I downgrade to a previous version.

**Comment by shilman** [comment]
@Nosherwan try this: https://storybooks.js.org/docs/react-storybook/basics/quick-start-guide/

If you use `getstorybook` it should install the old version (@kadira/storybook 2.3.5)

Let me know if that doesn't work for you!

**Comment by Nosherwan** [comment]
Unfortunately it didn't either.

**Comment by anton-tarasov-qs** [comment]
I can also confirm that the standard command
`npm install -g npm`
creates a broken storybook enviroment in which NPM can't install the storybook itself.

**Comment by NeonPaul** [comment]
It tries to install a version of @storybook/react that isn't published.

I manually added the actual published dependencies to my package.json, ran `npm run storybook` and everything worked fine.

``` json
{
 "devDependencies": {
    "@storybook/addon-actions": "^3.0.0-alpha.0",
    "@storybook/addon-links": "^3.0.0-alpha.0",
    "@storybook/react": "^3.0.0-alpha.0"
  }
}
```

**Comment by ajhyndman** [comment]
Maybe it's a bit confusing that the npm packages [`getstorybook`](https://www.npmjs.com/package/getstorybook) and [`@storybook/cli`](https://www.npmjs.com/package/@storybook/cli) both install a global script accessible via the same alias: `getstorybook`.

If you `npm uninstall -g @storybook/cli` and then `npm install -g getstorybook`, you should get version 2.35.3, as @shilman tried to explain above.

**Comment by aaronmcadam** [comment]
> Maybe it's a bit confusing that the npm packages getstorybook and @storybook/cli both install a global script accessible via the same alias: getstorybook.

Is that logged anywhere @ndelangen?

**Comment by emilong** [comment]
> If you npm uninstall -g @storybook/cli and then npm install -g getstorybook, you should get version 2.35.3

FYI check your package.json and remove any additions before rerunning `getstorybook`... That just bit me.

**Comment by danrashid** [comment]
What @emilong said, and I had to remove the .storybook dir

**Comment by Nosherwan** [comment]
Thanks everyone for the suggestions. @ajhyndman & @emilong's suggestions worked and I can revert now to getstorybook.

**Comment by ndelangen** [comment]
This should be fixed in `3.0.0-alpha.3`, let me know if you still run into this issue?
